test/functional/test-resources.js

```diff
@@ -1824,6 +1833,67 @@ describe('Resources mutateElement and collapse', () => {
     });
   });
 
+  it('attemptCollapse should not call attemptChangeSize', () => {
```

Comment:
but `#completeCollapse` gets called after `#getLayoutBox`. 
https://github.com/ampproject/amphtml/blob/master/src/service/resources-impl.js#L1029
Or should I reorder the call, as I feel the order doesn't matter here. It makes more sense to call `#changeSize` first and then measure layout box.